ui/helpers/utils/permission.js

```diff
@@ -14,16 +15,19 @@ import {
 } from '../../../shared/constants/permissions';
 ///: BEGIN:ONLY_INCLUDE_IN(flask)
 import { SNAPS_METADATA } from '../../../shared/constants/snaps';
-import { Icon, ICON_NAMES } from '../../components/component-library';
-import { coinTypeToProtocolName, getSnapDerivationPathName } from './util';
 ///: END:ONLY_INCLUDE_IN
+import { Icon, ICON_NAMES } from '../../components/component-library';
+import { Color } from '../constants/design-system';
+import { coinTypeToProtocolName, getSnapDerivationPathName } from './util'; // eslint-disable-line no-unused-vars
```

Comment:
[suggestion] I believe we could replace lines 18-21 w/

```
import { Icon, ICON_NAMES } from '../../components/component-library';
import { Color } from '../constants/design-system';
///: BEGIN:ONLY_INCLUDE_IN(flask)
import { SNAPS_METADATA } from '../../../shared/constants/snaps';
import { coinTypeToProtocolName, getSnapDerivationPathName } from './util';
///: END:ONLY_INCLUDE_IN
```
to avoid `// eslint-disable-line no-unused-vars`